pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -63,7 +84,7 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *api.Pod, podIP
 	cgroupParent := ""
 	portMappings := []*runtimeApi.PortMapping{}
 	for _, c := range pod.Spec.Containers {
-		opts, err := m.runtimeHelper.GenerateRunContainerOptions(pod, &c, podIP)
+		opts, err := m.runtimeHelper.GenerateRunContainerOptions(pod, &c, "")
```

Comment:
will add a todo for this. Do we deal the port name somewhere in today's docker runtime?